src/main/kotlin/com/autonomousapps/tasks/FindInlineMembersTask.kt

```diff
@@ -183,11 +183,21 @@ internal class InlineMembersFinder(
           // Guaranteed to be non-empty
           inlineMembers = inlineMembers
         )
-      }.toSortedSet().also {
+      }.groupByPackageName().also {
```

Comment:
Rather than this change, I think I'd prefer to fix this by fixing the `compareTo` function of `InlineMemberCapability.InlineMember`. Is that a change you're willing to take on? A useful example might be in how `Advice` implements that [here](https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/blob/main/src/main/kotlin/com/autonomousapps/model/Advice.kt#L24-L27). I believe that would be clearer and more understandable in the future, when we might forget why this change was made in the first place.

Reply:
Sure. I reverted my original fix and fixed `InlineMember.compareTo` impelementation. The only problem I don't know any built-in way to compare `Set` instances, so I also added custom `LexicographicIterableComparator` implementation. Please let me know if you know any better way to compare sets.